### DISCUSSION TITLE
Markdown cells do not intelligently complement latex code in jupyter extensions

### DISCUSSION THREAD

**Discussion by mainDevgithub** [microsoft/vscode-discussions#791] [Extension Development QnA] [answered]
I have installed the LaTex Workshop plugin with the LaTex plugin, and I have texlive installed on my computer with the environment variables set correctly. In VS Code, the .tex file is able to smart hint latex code completion, but not in the markdown cell of the .ipynb file. (Translation from DeepL)

**Comment by gjsjohnmurray** [comment, marked as answer]
Please raise this with the extension developers. The "Report Issue" option on VS Code's Help menu can get your report to the correct place. This forum is intended for extension developers to learn from one another.

**Comment by mainDevgithub** [comment]
OK



---Original---
From: "John ***@***.***&gt;
Date: Thu, Aug 31, 2023 22:54 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [microsoft/vscode-discussions] Markdown cells do notintelligently complement latex code in jupyter extensions (Discussion #791)




 
Please raise this with the extension developers. The "Report Issue" option on VS Code's Help menu can get your report to the correct place. This forum is intended for extension developers to learn from one another.

**Comment by olguzzar** [comment]
Closing as this question is off-topic for the forum, which is specifically for extension development.
